### DISCUSSION TITLE
Naxx Armor Transmogs from TBC Worldbosses

### DISCUSSION THREAD

**Discussion by AcetylGer** [chromiecraft/chromiecraft#4444] [Suggestions]
My suggestion is to add the T3 armor set transmogs into the TBC worldboss loot table.
You can in the future get the weapon transmogs from the pvp events but there is currently no way to get the transmogs for the armor sets. 
Killing the bosses every 3 days (if they work like the classic worldbosses) is going to be a good ammount of work for most guilds and it would be nice if the droprates could be adjusted so that most players can have their full T3 transmog before the lvl 70 bracket.
This would add another motivation to regularly killing the world bosses and we could even enable getting the actual T3 items from worldbosses once the lvl 70 bracket hits since naxx gear is at most pre raid bis for Karazhan.
Optional: add tuned hc versions of the worldbosses which drop T3 gear while the untuned worldbosses only drop the transmog. This would be in line with the untuned and hc raids coming with TBC.

**Comment by Peti321** [comment]
I think the TBC world bosses are harder than the Vanilla world bosses so this could be a great incentive for guilds to regularly kill them. This could be a great feature! I agree with the suggestion.

**Comment by Zelfie** [comment]
This sounds like a great PVE option to make the transmogs collectable.
IIRC they tied some Tier 3 transmogs to a custom PVP event that has a very limited uptime. So it'd be nice for a viable PVE option, that can be done any time, for our cross-world guilds or PVP-avoiders.

**Comment by Cooties-cc** [comment]
There still hasn't been much word on when they plan to script the TBC world bosses, even though they were added at launch, I assume they would be fine when t4 launches. There was also talks about adding a token-esk system with aq40 bosses as bonus loot, where the t3 mogs could be obtained. However it was shot down because it was implied that an additional MPQ would have to be added if they were just transmog items (no stats, etc). Yet the pvp event was able to allow these mogs to exist with no MPQ download, so it begs the question why it couldn't be done in aq40. 
If a vendor based system is the easier way to implement these items, over changing a loot table on a mob, then we could also see if devs are willing to add the mogs to the badge of justice vendors. It would incite more heroics to be ran, more bugs to be found, help other players overall to do more content, and doesn't hurt any game systems or decline normal players access to mogs. 

Overall the idea to add in t3 mogs should be considered since the server allows xmogs, and some naxx60 item models were unique only to that raid. I think there's a lot of "spare" content that could easily be implemented at a low dev cost which would benefit the server.

**Comment by balleny** [comment]
Imo, if possible the t3 tmog should just be added to everyones collection by default so this repeating discussion comes to an end :)

**Reply by balleny** [reply]
Adding it to a shop / drop table will make it more tricky, cause only the rings are class-restricted

**Reply by BeefSteaker** [reply]
Then add the tokens + scraps to drop tables. 

Quest handin npc's are outside of the instance unlike 2.5

**Comment by BeefSteaker** [comment]
I really like the transmogs so any time before the naxx80 recolours would be great

**Comment by shaszun** [comment]
I think it's best for the server these to be added to the shop, it will give the server some revenue to keep going. Not only the sets, but also the weapons too.

**Comment by Amrx2** [comment]
Hi all, T3 transmogs is in the pipeline, we are focusing on the TBC launch now but will be revisitied during the TBC brackets.

Just popping this message on here just to confirm its not been missed.

Have a good day,
Construct
